ria_remote/resources/procedures/ria_post_install.py

```
@@ -0,0 +1,103 @@
+    if not base_path:
+        lgr.error(
+            'ria_post_install logic error: active RIA special '
+            'remote without base-path setting')
+        return
+    dspath_in_ria = (base_path, ds.id[:3], ds.id[3:])
+    if not (origin_remote['url'].endswith(posixpath.join(*dspath_in_ria)) or
+            origin_remote['url'] == os.path.join(*dspath_in_ria)):
+        # we have no business here. neither a local, nor a remote clone
+        # from this RIA store
+        return
+
+    # we don't want annex copy-to origin
+    ds.config.set(
+        'remote.origin.annex-ignore', 'true',
+        where='local')
+
+    ds.siblings(
+        'configure',
+        name='origin',
+        publish_depends=rm['name'],
+        result_filter=None,
+        result_renderer='disabled')
+
+    if ephemeral:
+        # with ephemeral we declare 'here' as 'dead' right away, whenever
+        # we symlink origins annex. Because we want annex to copy to
+        # the ria remote to get availability info correct for an eventual
+        # git-push into the store
+        # this will cause stuff like this for a locally present annexed file:
+        # % git annex whereis d1
+        # whereis d1 (0 copies) failed
+        # BUT this works:
+        # % git annex find . --not --in here
+        # % git annex find . --in here
+        # d1
+        ds.repo._run_annex_command('dead', annex_options=['here'])
+
+        if reckless and origin_remote['url'] == os.path.join(*dspath_in_ria):
+            # cloned from a RIA store at a local path, symlink the annex
+            # to avoid needless copies in an emphemeral clone
+            annex_dir = ds.repo.dot_git / 'annex'
+            rmtree(str(annex_dir)) \
+                if not annex_dir.is_symlink() else annex_dir.unlink()
+            annex_dir.symlink_to(
+                os.path.join(origin_remote['url'], 'annex'),
+                target_is_directory=True)
+
```

If `reckless` was given but w/o `ephemeral` we silently ignore it.
Same for `reckless` but it`s not local.
Both should prob. give at least a message.